homeassistant/components/weatherflow_cloud/sensor.py

```diff
@@ -34,6 +46,104 @@ class WeatherFlowCloudSensorEntityDescription(
     value_fn: Callable[[Observation], StateType | datetime]
 
 
+@dataclass(frozen=True, kw_only=True)
+class WeatherFlowCloudSensorEntityDescriptionWebsocketWind(
+    SensorEntityDescription,
+):
+    """Describes a weatherflow sensor."""
+
+    value_fn: Callable[[EventDataRapidWind], StateType | datetime]
+
+
+@dataclass(frozen=True, kw_only=True)
+class WeatherFlowCloudSensorEntityDescriptionWebsocketObservation(
+    SensorEntityDescription,
+):
+    """Describes a weatherflow sensor."""
+
+    value_fn: Callable[[WebsocketObservation], StateType | datetime]
+
+
+WEBSOCKET_WIND_SENSORS: tuple[
+    WeatherFlowCloudSensorEntityDescriptionWebsocketWind, ...
+] = (
+    WeatherFlowCloudSensorEntityDescriptionWebsocketWind(
+        key="wind_speed",
+        translation_key="wind_speed",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.WIND_SPEED,
+        suggested_display_precision=1,
+        value_fn=lambda data: data.wind_speed_meters_per_second,
+        native_unit_of_measurement=UnitOfSpeed.METERS_PER_SECOND,
+    ),
+    WeatherFlowCloudSensorEntityDescriptionWebsocketWind(
+        key="wind_direction",
+        translation_key="wind_direction",
+        state_class=SensorStateClass.MEASUREMENT,
+        value_fn=lambda data: data.wind_direction_degrees,
+        native_unit_of_measurement="°",
+    ),
+    WeatherFlowCloudSensorEntityDescriptionWebsocketWind(
+        key="websocket_wind_epoch",
+        translation_key="websocket_wind_epoch",
+        state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        device_class=SensorDeviceClass.TIMESTAMP,
+        suggested_display_precision=1,
+        value_fn=(
+            lambda data: datetime.fromtimestamp(data.epoch, tz=UTC)
+            if data.epoch is not None
+            else None
+        ),
+    ),
```

Comment:
What does this display?

Reply:
I mean, it's a websocket, and thus push based, so the entities that are listening to these are only updated when the websocket updated, so no need for a separate state